taichi/gui/gui.h

```diff
@@ -888,7 +888,7 @@ class TI_DLL_EXPORT GUI : public GUIBase {
       redraw();
       process_event();
       if (frame_id % 10 == 0)
-        set_title(fmt::format("{} ({:.2f} FPS)", window_name, real_fps));
+        set_title(fmt::format("{} ({:.2f} FPS)", "", real_fps));
```

Comment:
Why's this change?

Reply:
Accidentally included. The text on my Windows 10 is all garbled, tried to fix it but no luck yet